### PR TITLE
Reformat ovn-heater with Black

### PR DESCRIPTION
**Pull request by almusil**
Reformat the codebase with Black
and add it to CI runs. To format
the files locally you can run:
black -l 79 -S $FILE

Signed-off-by: Ales Musil <amusil@redhat.com>

### CODE REVIEW

**Comment by LorenzoBianconi** [comment]
@almusil I am fine with this change, but I think we should postpone it after IPv6 support

**Comment by almusil** [comment]
Definitely, that's why it is draft, I don't want to cause rebase pain :)

**Comment by dceara** [comment]
@almusil the IPv6 support is merged.  Do you plan to rebase this PR and turn it into a formal one?

**Comment by putnopvut** [comment]
+1 on getting this merged soon. I tend to have my python files auto-formatted with Black when I save them. I currently have to disable that with ovn-heater since it results in huge diffs otherwise.

**Comment by almusil** [comment]
Rebased

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -23,7 +23,11 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install flake8
+        python -m pip install flake8 black==22.3.0
+    # Ignore E203, W503 and W504 which is against PEP 8 style
     - name: flake8
       run: |
-        flake8 ovn-tester
+        flake8 --ignore=E203,W504,W503 .
```
Comment (dceara) [inline .github/workflows/test.yml:30]: This now runs flake8 on more files which is a good thing.  But we should also fix the warnings it raises in those files, e.g.:
```
./ovn-fake-multinode-utils/process-monitor.py:51:9: E722 do not use bare 'except'
./utils/process-stats.py:2:1: F401 'numpy' imported but unused
./utils/process-stats.py:7:1: F401 'time' imported but unused
```

Comment (almusil) [reply .github/workflows/test.yml:30]: Ah didn't see those as the CI was blocked, ack I will address it

**Review by dceara — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**